lib/addons/p5.dom.js

```diff
@@ -550,12 +555,20 @@
    * }
    * </code></div>
    */
-  p5.prototype.createSelect = function(mult) {
-    var elt = document.createElement('select');
-    if (mult){
-      elt.setAttribute('multiple', 'true');
+
+  p5.prototype.createSelect = function() {
+    var elt,self;
+    var arg = arguments[0];
+    if( typeof arg === 'object' && arg.elt.nodeName == 'SELECT' ){
```

Comment:
This could create an error if an object other than a p5.Element is passed in. Instead this check could be:
if( arg.elt && arg.elt.nodeName == 'SELECT' ){